xbmc/pvr/guilib/guiinfo/PVRGUIInfo.cpp

```diff
@@ -428,6 +428,13 @@ bool CPVRGUIInfo::GetListItemAndPlayerLabel(const CFileItem* item, const CGUIInf
         strValue = recording->IsRadio() ? m_strPlayingRadioGroup : m_strPlayingTVGroup;
         return true;
       }
+      case VIDEOPLAYER_YEAR:
+      case LISTITEM_YEAR:
+        if (recording->GetPremiered().IsValid())
```

Comment:
Why use `recording->GetPremiered()` instead of `recording->GetYear()`?

Reply:
NVM, ignore this question.

Reply:
> 
 > Why use `recording->GetPremiered()` instead of `recording->GetYear()`?

In this case recording is sub class of CVideoInfoTag so GetYear() will return First Aired by preference.  I suppose I could add a GetYear() to CPVRRecording if that is more logical.

Reply:
That's ok, that is essentially what I was suggesting, to override GetYear()

Reply:
... override GetYear() and HasYear().

Reply:
Ok that doesn't seem to have any issues.  Should the code continue to populate the year into first aired or should that be reserved for the normal case meaning first aired on TV often not the same year.  The skin designer would still have the choice of adding it.

![image](https://user-images.githubusercontent.com/2148031/93524511-95ff2a80-f902-11ea-8aa4-9117b9d537cf.png)

Reply:
Can you reference where year is populated into first aired? I'm not seeing that currently?

Reply:
> 
> 
> Can you reference where year is populated into first aired? I'm not seeing that currently? Or do you mean in PVRGUIInfo?

Yes.  However if PVRGUIInfo was changed to process only first aired, the case would also need to return true or VideoGUIInfo could still fill in the year.
> 
> If that is what you mean I think it is ok as it's a sensible alternative if first aired is not provided.

My feeling is if the addon wanted to provide the alternative it could.  That seems better than having forced, potentially incorrect data that can't be skinned.